CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ The format is based on [Keep a Changelog], and this project adheres to
 
 ## [Unreleased]
 
+## Changed
```

Comment:
```suggestion
### Changed
```

Sorry, I messed up when I pasted this in before. 

Also, just to be hyper-pedantic can you also please move this section below `Added`. For future reference there's a "template" of sorts that shows you the order at the [bottom of each changelog](https://github.com/dogmatiq/marshalkit/blame/893dbf48738d9c6d4a774e357645c0c4d29aaf8e/CHANGELOG.md#L72).

Reply:
All good I force pushed the change to keep the commit history slightly nicer.

Reply:
Order of the sections is still the same.

Reply:
Ah sorry, here comes another force-push...